File: OOP/something/something.py

class Person(object):

    def __init__(self, name, age):
        self.name = name
        self.age = age

    @staticmethod
    def print_name(self):
        print 'Helloka'


if __name__ == '__main__':
    p = Person('Liana', 23)
    Person.print_name()


class CommandExecuter(object):

    @staticmethod
    def run_command(command_text):
        print 'Hello, I ran {something}'.format(something=command_text)
        clt.execute(command_text)

CommandExecuter.run_command()

File: OOP/something/something.java


public class Person {

    public String name;
    public int age; 

    public Person(String name, int age) {
        this.name = name;
        this.age = age;
    }


    public void sayName() {
        println(name);
    }

    public static void sayLatinName() {
        println('Homo Sapiens');
    }

}

Person p = new Person('Liana', 23);
p.sayName(); // Liana
p.sayLatinName(); // Homo Sapiens

Person.sayLatinName(); // Homo Sapiens


